test/parse-outputs/test_parse_outputs.py

```diff
@@ -89,3 +92,12 @@ def test_duplicate_edges(self):
 
             runner.parse_output(algo, test_file, out_file, params)
             assert filecmp.cmp(OUTDIR / f"{algo}-duplicate-pathway.txt", EXPDIR / f"{algo}-pathway-expected.txt", shallow=False)
+
+    def test_empty_raw_pathway(self):
+        for algo, params in algorithms.items():
+            files = glob(str(INDIR / "empty" / f"{algo}-empty-raw-pathway*"))
+            assert len(files) != 0, "can't test on no empty raw pathways!"
+            for test_file in glob(str(INDIR / "empty" / f"{algo}-empty-raw-pathway*")):
+                out_file = OUTDIR / f"{algo}-empty-raw-pathway-output.txt"
```

Comment:
This output file should be `f"{algo}-empty-pathway-output.txt"` since it is converting the raw-pathway into the standarized pathway formart